src/hooks/date.ts

```diff
@@ -0,0 +1,33 @@
+import moment from 'moment';
+
+// 날짜 형식을 변경하는 함수
```

Comment:
이런 util 류는 input, output 예시를 위에 주석으로 적어주시면 가독성이 매우 좋아져요!
```suggestion
// 날짜 형식을 변경하는 함수 new Date -> 2024.05.07
```